_data/changelogs/component-date-picker.yml

```
@@ -2,6 +2,14 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Adds focus styles to calendar button in high contrast mode. 
+    summaryAdditional: Now, the calendar icon changes to the `highlight` high contrast token.
```

Suggesting a couple small changes: Update to past tense and add detail about when the icon changes
```suggestion
    summary: Added focus styles to calendar button in high contrast mode. 
    summaryAdditional: Now, the calendar icon changes to the `highlight` high contrast token on focus.
```